### PR TITLE
[KIALI-902] Improve itest bot to fetch images on demand

### PR DESCRIPTION
**Pull request by aljesusg**
itest only run if there is a label like: 

- kiali-bot-graph
- kiali-bot-services
- kiali-bot-services-review
- kiali-bot-reviews-metrics 
- kiali-bot-istio
- kiali-bot-istio-promhttp-rules 
- kiali-bot-jaeger

It's working with a regular expression so, if you add for example labels:

- kiali-bot => execute all tests
- kiali-bot-istio => kiali-bot-istio & kiali-bot-istio-promhttp-rules

### CODE REVIEW

**Comment by aljesusg** [comment]
You should run again travis ^^ or....create a bot.

About the second question...it's more cleaner labels I think and you don't need to check the comment. I don't know...maybe we can check how to do it...this is an idea for solve the noise and improve this.

File: run_itest.sh [hunk 1]
```diff
@@ -12,9 +12,19 @@ APP_PID=$!
...
+mkdir -p itest_img
 
-yarn itest
+echo "Getting labels of https://api.github.com/repos/kiali/kiali-ui/pulls/${TRAVIS_PULL_REQUEST}"
+
+RAW_DATA=$(curl -s https://api.github.com/repos/kiali/kiali-ui/pulls/${TRAVIS_PULL_REQUEST})
+IFS=',' read -r -a labels <<< $(python2 getLabels.py "${RAW_DATA}")
+for label in "${labels[@]}"
+do
+    yarn itest -t "${label}"
```
Comment (josejulio) [inline run_itest.sh]: Isn't there any option to pass multiple labels?
e.g. yarn itest -t labels, labels, labels

Comment (aljesusg) [reply run_itest.sh]: I tried to pass a list or use other command but not...I need to investigate more about arguments

**Comment by vnugent** [comment]
+1 on expanding the test to other screen.  I'm testing out posting the screenshot as a Review comment instead of regular comment to see if that will not trigger email notification. (KIALI-898)

**Comment by aljesusg** [comment]
Great thanks @lucasponce

**Pull request merged**